queries/orderbook/barn_batch_rewards.sql

```diff
@@ -63,19 +84,22 @@ order_surplus AS (
         ON s.auction_id = ss.auction_id
         JOIN trades t -- contains traded amounts
         ON s.block_number = t.block_number -- log_index cannot be checked, does not work correctly with multiple auctions on the same block
-        JOIN orders o -- contains tokens and limit amounts
-        ON t.order_uid = o.uid
+        JOIN order_data od -- contains tokens and limit amounts
+        ON t.order_uid = od.uid
         JOIN order_execution oe -- contains surplus fee
         ON t.order_uid = oe.order_uid
         AND s.auction_id = oe.auction_id
         LEFT OUTER JOIN order_quotes oq -- contains quote amounts
-        ON o.uid = oq.order_uid
+        ON od.uid = oq.order_uid
         LEFT OUTER JOIN app_data ad -- contains full app data
-        on o.app_data = ad.contract_app_data
+        on od.app_data = ad.contract_app_data
     WHERE
-        ss.block_deadline >= {{start_block}}
-        AND ss.block_deadline <= {{end_block}}
+        ss.block_deadline >= 20759465
```

Comment:
probably leftover from local testing

Reply:
Good catch. That will hopefully explain the small discrepancy in total protocol fees.